libevmasm/Instruction.cpp

```diff
@@ -174,6 +175,7 @@ std::map<std::string, Instruction> const solidity::evmasm::c_instructions =
 	{ "SELFDESTRUCT", Instruction::SELFDESTRUCT }
 };
 
+/// @note InstructionInfo is assumed to be the same across all EVM versions.
```

Comment:
Is that so :-)? The PR just changed this, didn't it :-)?
```suggestion
/// @note InstructionInfo is assumed to be the same across all EVM versions except for the instruction name.
```

Reply:
Maybe better even to be precise about it - i.e. it currently assumes that args, ret, and side effects are constant, right?

Reply:
done.